.eslintrc.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
         ],
         'key-spacing': ['error', { beforeColon: false, afterColon: true }],
         'keyword-spacing': ['error', { before: true, after: true }],
-        'linebreak-style': ['error', 'unix'], // no carriage returns
+        'linebreak-style': 0, // no carriage returns
```

Comment:
Please don't change this, we do not want carriage returns. Make sure your editor is correctly configured with editorConfig

Reply:
I understand. In regards to this change, I've changed my end of line character to \n so that it is for the unix type, and the linter will return 1100+ errors in regards to the spacing. yarn lint --fix does not autofix anything, neither does the selction 'fix all line space errors'. 

Would you be able to offer any suggestions?

Reply:
![linter](https://user-images.githubusercontent.com/99618731/224257131-b931283a-ec3c-4970-bf19-3aa0dd1da6f1.png)

I just changed it back to that error unix property and ran yarn lint --fix command

Reply:
I manually fixed all the errors. The linting should pass at least. Still, I'm not sure why the auto fix functions don't work...

Reply:
On our readme we note that everyone should be using the https://editorconfig.org/ tool, and if you use VSCode, you will need to install a special plugin. This file will set up expected line endings and formatting

🤔 Can we make this important setup step clearer somehow? Maybe we need to highlight it in CONTRIBUTING.md

Git on windows sometimes does sneaky things, replacing `\n` with `CRLF` on checkout, and converting back on upload.

Reply:
Hmm, I have the editorconfig extension installed on my vs code and I have the editorconfig file pulled. That is interesting about the \n for git on windows.